PostgresV11.md

```diff
@@ -85,46 +65,8 @@ select * from event;
 
 **The following steps should be done to define the micro-services dependency to the new V11 DB container**
 
-* Open backend.yml file and uncomment the dependency to ccd-shared-database-v11 for 'definition-store' and 'data-store'. 
-```$xslt
-#Uncomment this line to enable ccd-shared-database with Postgres version 11
-      ccd-shared-database-v11:
-        condition: service_started
-```
-* Comment the dependency to ccd-shared-database for 'definition-store' and 'data-store' on backend.yml file.
-```$xslt
-#      ccd-shared-database:
-#        condition: service_started
-```
-* Open message-publisher.yml file and uncomment the dependency to ccd-shared-database-v11.  
-```$xslt
-#Uncomment this line to enable ccd-shared-database with Postgres version 11
-      ccd-shared-database-v11:
-        condition: service_started
 ```
-* Comment the dependency to ccd-shared-database on message-publisher.yml file.
-```$xslt
-#      ccd-shared-database:
-#        condition: service_started
-```
-
-* Add CCD_POSTGRES_11 env var to your local terminal bash file.
-
-* Export CCD_POSTGRES_11=ccd-shared-database-v11 in your terminal. 
-```$xslt
- export CCD_POSTGRES_11=ccd-shared-database-v11
-```
-* Uncomment CCD_POSTGRES_11 in your .env file.
-````
-#Postgres V11
-CCD_POSTGRES_11=ccd-shared-database-v11
-````
-
-1) Open a new terminal, make sure that CCD_POSTGRES_11 environment variable has been set.
-```
-env | grep CCD_POSTGRES_11
-```
-2) Stop and re-start the ccd-docker.
+1) Stop and re-start the ccd-docker.
    
```

Comment:
minor correction: In this section, texts are highlighted rather than the script/code. It should be other way.